ERCS/erc-6932.md

```diff
@@ -0,0 +1,122 @@
+---
+eip: 6932
+title: Subscription-Based Token
+description: ERC-20 extension providing access to a service or product that requires recurring payments.
+author: 360 Core <hello@360coreinc.com>, Robin Rajput (@0xRobinR)
+discussions-to: https://ethereum-magicians.org/t/erc-subscription-based-erc20-token/13964
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-04-25
+requires: 20
+---
+
+## Abstract
+
+The subscription-based [ERC-20](./eip-20.md) token extends the basic [ERC-20](./eip-20.md) token standard with a `subscribe` and `unsubscribe` function, which allow users to subscribe or unsubscribe from the subscription service. The `subscriptionFee` and `subscriptionFrequency` variables define the cost and frequency of the subscription. The `nextPaymentDate` mapping keeps track of the next payment date for each subscriber.
+
+This EIP proposes the standardization of a subscription token on the Ethereum blockchain, enabling automatic periodic deductions from user balances as determined by the merchant subscriber. The Subscription Token aims to simplify and streamline subscription-based services on the Ethereum network, offering enhanced convenience and efficiency for users and merchants alike.
+
+We will be also adding `renewSubscription` functions to the token contract, that can be used by token holders to renew their subscription to a service or product that requires recurring payments in the form of the token.
```

Comment:
You should write proposals as if they were already in the Final status. This line won't make sense once this proposal is Final.